src-vue/src/views/DeveloperView.vue

```diff
@@ -308,6 +320,21 @@ export default defineComponent({
                     notification.close();
                 });
         },
+        async installNSProton() {
+            await invoke("install_northstar_proton_wrapper")
+                .then((message) => { showNotification(`Done`); })
+                .catch((error) => { showNotification(`Error`, error, "error"); })
+        },
```

Comment:
My internet connection is so slow right now that I cannot test installation atm cause the download takes forever. My takeaway from this however is that it might make sense to spawn an initial notification telling the user that installation has begun and then clearing it again afterwards on success/failure.

Example reference:
https://github.com/R2NorthstarTools/FlightCore/blob/main/src-vue/src/views/DeveloperView.vue#L255-L268


Optimally we should have a progress bar but that's kinda outside the scope of the PR and requires changes upstream.

Reply:
Sure, I can add a little notification stating "Started Proton Install"

This is basically just a minimally viable product, in the future this will be fleshed out and have all the libthermite stuff removed.